### ISSUE TITLE
NVDA fails to report list items in Explorer, needs to restart Explorer

### ISSUE THREAD

**Issue by cary-rowen** [nvaccess/nvda#14453]
This Issue has not yet found a stable way to reproduce it.
### Steps to reproduce:
Use the up and down arrows to navigate in Explorer's file list after the problem occurs.
### Actual behavior:
NVDA should report the name of the currently focused list item.
### Expected behavior:
NVDA doesn't report anything
### NVDA logs, crash dumps and other attachments:
[log.txt](https://github.com/nvaccess/nvda/files/10246021/log.txt)

#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
NVDA2022.4Beta3 
#### Windows version:
Windows 10 22H2 (x64) build 19045.2364
#### Name and version of other software in use when reproducing the issue:
None
#### Other information about your system:
None
#### Does the issue still occur after restarting your computer?
No
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
This issue also occurs in older versions of NVDA
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes, But the tool requires restarting the computer, after restarting the computer the problem disappeared.

**Comment by michaelDCurran** [comment]
For now there is nothing we can do here as it is impossible to reproduce. However, if it occurs again, please set your NVDA log level to debug, enable UIA debug logging in NVDA's advanced settings, reproduce the issue, provide the log here or privately, and reopen the issue.

**Comment by cary-rowen** [comment]
@michaelDCurran 
I really didn't find a stable way to reproduce it, but I captured the logs as you asked.
The problem occurs when I try to open the NVDA config folder, but this should not be the way to reproduce it stably.
[log_WithUIAAndMSAA.txt](https://github.com/nvaccess/nvda/files/10287703/log_WithUIAAndMSAA.txt)

**Comment by Adriani90** [comment]
@cary-rowen are you still having this issue in NVDA 2023.3?

**Comment by Adriani90** [comment]
Or even better, can you test with the last NVDA alpha?

**Comment by cary-rowen** [comment]
Which PR do you think would help fix this issue?

**Comment by Adriani90** [comment]
Oh there could be alot since this issue has been opened, or Windows updates or whatever. In case this is not an issue anymore, it makes sense to close it.

**Comment by beqabeqa473** [comment]
It is still reproduceable and it is not possible to give actual str

On 11/18/23, Adriani90 ***@***.***> wrote:
> Oh there could be alot since this issue has been opened, or Windows updates
> or whatever. In case this is not an issue anymore, it makes sense to close
> it.
>
> --
> Reply to this email directly or view it on GitHub:
> https://github.com/nvaccess/nvda/issues/14453#issuecomment-1817473128
> You are receiving this because you are subscribed to this thread.
>
> Message ID: ***@***.***>


-- 
with best regards Beqa Gozalishvili
Tell: +995593454005
Email: ***@***.***
Web: https://gozaltech.org
Skype: beqabeqa473
Telegram: https://t.me/gozaltech
facebook: https://facebook.com/gozaltech
twitter: https://twitter.com/beqabeqa473
Instagram: https://instagram.com/beqa.gozalishvili

**Comment by cary-rowen** [comment]
Yes, as @beqabeqa473 said, this is still a reproducible issue. But it's hard to give clear steps.

**Comment by cary-rowen** [comment]
Preview pane not enabled, details enabled. But I don't think that's relevant, sometimes I also use list views. Thanks.

**Comment by Adriani90** [comment]
@cary-rowen, @beqabeqa473 are you also having this issue in Windows 11 with NVDA 64 bit last alpha?

**Comment by tareh7z** [comment]
Hi!  

I can confirm that this issue exists and is still reproducible.  

Thanks.

**Comment by Adriani90** [comment]
Can you please add the exact version of Windows version you use? Did you test with NVDA 64bit alpha?Von meinem iPhone gesendetAm 06.11.2025 um 01:12 schrieb Tarik Hadžirović ***@***.***>:﻿hdzrvcc0X74 left a comment (nvaccess/nvda#14453)
Hi!
I can confirm that this issue exists and is still reproducible.
Thanks.

**Comment by tareh7z** [comment]
Hi!  

I’m using Windows 11 25H2 (AMD64) build 26200.6899 and [NVDA alpha-53273,a6a536eb](https://download.nvaccess.org/snapshots/alpha/nvda_snapshot_alpha-53273,a6a536eb.exe).  

Thanks.
